tests/test_optimizer.py

```diff
@@ -1414,7 +1414,8 @@ def test_optimize_coil_currents(DummyCoilSet):
         verbose=2,
         copy=True,
     )
-    # check that optimized coil currents changed by more than 15% from initial values
+    # check that on average optimized coil currents changed by more than
```

Comment:
#1406 made the same change I used here, the change of comment is from my version.